clang/test/compiler-course/kozlova_e_var_4_test/test.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/FirstPlugin_KozlovaEkaterina_FIIT3_ClangAST%pluginext -plugin FirstPlugin_KozlovaEkaterina_FIIT3_ClangAST -fsyntax-only %s 2>&1 | FileCheck --match-full-lines %s
+
+
+// CHECK: int global_var1 = 0;
+// CHECK-NEXT: int foo(int param_a, int param_b) {
+// CHECK-NEXT:   static int static_var2 = 0;
+// CHECK-NEXT:   int local_var3 = 123;
+// CHECK-NEXT:   ++static_var2;
+// CHECK-NEXT:   return param_a + param_b + global_var1 + static_var2 + local_var3;
+// CHECK-NEXT: }
+// CHECK-NEXT: int static global_var4 = 1;
+//CHECK: int static global_var5 = foo(1, global_var4);
+
+
+int var1 = 0;
+int foo(int a, int b) {
```

Comment:
Could you please extend name of a parameter var with default value if present. I.e.
```suggestion
// CHECK-NEXT: int foo(int param_a, int param_b_default_10) {
int foo(int a, int b = 10) {
```